src/app/nation/nation.component.html

```diff
@@ -16,11 +16,11 @@ <h1 i18n>Nation List</h1>
       </thead>
       <tbody>
         <ng-container *ngFor="let nation of nations; let i = index">
-        <tr *ngIf="nation.doc.name.toLowerCase().includes(searchText)">
-          <td>{{nation.doc.name}}</td>
-          <td>{{nation.doc.admin_name}}</td>
-          <td>{{nation.doc.nationurl}}</td>
-        </tr>
+          <tr *ngIf="nation.doc.name.toLowerCase().includes(search_text) || nation.doc.name.toUpperCase().includes(search_text) || nation.doc.name === search_text">
```

Comment:
I think you only need the `toLowerCase()` part, but you should also do `searchText.toLowerCase()`